app/jobs/hackathons/send_digests_job.rb

```diff
@@ -22,6 +22,6 @@ def current_subscribers
   end
 
   def new_digest_pertinent?(subscriber)
-    subscriber.digests.last&.created_at&.before? 6.days.ago
+    !!subscriber.digests.last&.created_at&.before?(6.days.ago)
```

Comment:
It's alright if this method returns `nil`. no need to force it to be a boolean.

To address https://github.com/hackclub/hackathons-backend/pull/150#discussion_r1298821542 this method needs to return `true` if there are no digests (`return true if subscriber.digests.empty?`).

At the moment, it returns `nil` if `subscriber.digests.empty?`